chains/orchestrator-relays/runtime/dancelight/src/tests/inactivity_tracking.rs

```diff
@@ -81,6 +83,19 @@ fn get_collators_set(
     collator_set
 }
 
+fn get_chains_set(
+    chains: Vec<ParaId>,
+) -> BoundedBTreeSet<ParaId, <Runtime as pallet_inactivity_tracking::Config>::MaxContainerChains> {
+    let mut chains_set: BoundedBTreeSet<
+        ParaId,
+        <Runtime as pallet_inactivity_tracking::Config>::MaxContainerChains,
+    > = BoundedBTreeSet::new();
+    chains.iter().for_each(|collator| {
+        chains_set.try_insert(*collator).ok();
```

Comment:
Since this is for tests, panic on error.

```rust
for para_id in chains {
    chains_set.try_insert(para_id).unwrap()
}
```

Reply:
Sure updated in https://github.com/moondance-labs/tanssi/pull/899/commits/ec3d31e2ccfed1f1b4406b768d22b0cf94b2c131